index.html

```diff
@@ -0,0 +1,162 @@
+            <p class="desk-about-par">The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+                This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge.
+                So if you're active and engaged in the worlds of open content and knowledge - free software advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together.</p>
+            <div class="contact-us">Please contact us per Email for any further questions about The Youth Submit 2023!</div>
+            <div class="email">youthsummit2023@gmail.com</div>
+    </section>
+    <div class="logo-explanation">Youth Leadership Summit 2023 Logo</div>
+         <hr class="hr1 hr21">
+         <div class="explanation">The logo of Youth Leadership Summit 2023 was decided through the logo competition from 8, June to 7, July.</div>
+         <p class="explanation2">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+            After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+            After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:</p>
+         <div class="summit"><img src="images/images/about_logo 1.png" alt=""></div>
+         <div class="summit-logo"><img class="img-fluid" src="images/images/about_logo 1 (1).png" alt=""></div>
+         <hr class="hr3 hr33">
+         <hr class="hr3">
+         <div class="past-summits">See the past Youth Summits</div>
+         <hr class="hr101">
+
+         <div class="look-in-past">Take a look at the past two Youth summits which took place in Moscow and Canada.</div>
+         <ul class="pictures">
+            <li><div class="group"><img class="img-fluid" src="images/images/about_2011 1.png" alt=""></div></li>
+            <li><div class="group-of-3"><img class="img-fluid" src="images/images/about_2013 1.png" alt=""></div></li>
+         </ul>
+         <footer>
+            <div class="partners">Partner
+                <ul class="partner-list">
+             <li><img src="images/images/mozilla 1.png" alt=""></li>
+                    <li><img src="images/images/googlekorea 1.png" alt=""></li>
+                    <li><img src="images/images/naver 1.png" alt=""></li> 
+                    </ul>
+                    <ul class="partner-list partner-list2">
+                    <li class="block"><img src="images/images/daumkakaologo 1.png" alt=""></li>
```

Comment:
As requested by the previous code reviewer this is a required change and it need to be worked on. It would be nice if you corrected this inorder for you to get an approval. Please pay attention to detail